.github/workflows/rust.yml

```
@@ -41,6 +41,133 @@ jobs:
+  workspace-tests:
+    name: Workspace Tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        rust: [stable, beta, nightly, 1.89.0]
+    steps:
+      - name: Checkout Crate
+        uses: actions/checkout@v4
+      - name: Setup Rust toolchain
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)